src/cart/crt_swim.c

```diff
@@ -1057,7 +1070,7 @@ static int64_t crt_swim_progress_cb(crt_context_t crt_ctx, int64_t timeout_us, v
 		 * The max_delay should be less suspicion timeout to guarantee
 		 * the already suspected members will not be expired.
 		 */
-		if (csm->csm_alive_count > 2) {
+		if (csm->csm_alive_or_suspect_count > 2) {
```

Comment:
Have you figured out that why it's okay for us to not update/extend the suspecting timeout if the number of alive_or_suspect_count is less or equal than 2? I tend to think it's applicable no matter how many ranks there are.

Reply:
@jxiong, please see

- "The number must be > 2 here perhaps because..." in https://github.com/daos-stack/daos/pull/15924/files#r1961102361 and
- "Moreover, not counting SUSPECT members..." in https://github.com/daos-stack/daos/pull/15945#issue-2865464360 (i.e., the description of this PR).

That is as far as I can figure out at the moment. Personally, I think `> 1` might be enough, for the 2-member case seems not that important. I kept `> 2`, however, until we find some stronger justification to change this condition.